src/main/java/cam72cam/immersiverailroading/tile/TileRail.java

```diff
@@ -130,6 +130,9 @@ public NBTTagCompound writeToNBT(NBTTagCompound nbt) {
 	public void setDrops(List<ItemStack> drops) {
 		this.drops = drops;
 	}
+	public List<ItemStack> getDrops () {
```

Comment:
Remove this function and change spawnDrops to take a location (can add a spawnDrops function that takes no args and calls spawnDrops with this.pos).